lgt8f/cores/lgt8f/main.cpp

```diff
@@ -102,10 +102,6 @@ void lgt8fx8x_init()
 // store ivref calibration value (According to the LGT8F328D databook, if VCAL is rewritten, the value of VCAL1 can be lost.)
 	_VCAL_1_ = VCAL1;
 
-// enable 1KB E2PROM 
```

Comment:
@LaZsolt The EEPROM.h library will set this to 1 by default when imported anyway.

If the lib is not imported, the ECCR register will stay at zero, which means "I won't use the EEPROM"

Reply:
Incompatible idea.